src/util/irep.cpp

```diff
@@ -13,7 +13,6 @@ Author: Daniel Kroening, kroening@kroening.com
 
 #include <ostream>
 
-#include "invariant.h"
```

Comment:
This was deleted because the corresponding header already includes `invariant.h`. I'm not a huge fan of this change myself, but it's not harmful.